app/commands/iteration/create.rb

```diff
@@ -67,13 +67,13 @@ def init_services
       [
         Thread.new do
           Iteration::TestRun::Init.(iteration_uuid, solution.track.slug, solution.exercise.slug, s3_uri)
+        end,
+        Thread.new do
+          Iteration::Analysis::Init.(iteration_uuid, solution.track.slug, solution.exercise.slug, s3_uri)
```

Comment:
Test pls.

Reply:
It looks like there already is a test for this: https://github.com/exercism/v3-website/blob/master/test/commands/iteration/create_test.rb#L22